inc/class-customizer-guided-tour.php

```diff
@@ -0,0 +1,213 @@
+
+			wp_enqueue_script( 'wc-api-dev-guided-tour', WC_API_Dev::$plugin_url . 'assets/js/admin/customizer' . $suffix . '.js', array( 'jquery', 'wp-backbone' ), $storefront_version, true );
+
+			wp_localize_script( 'wc-api-dev-guided-tour', '_wpCustomizeWcApiDevGuidedTourSteps', $this->guided_tour_steps() );
+		}
+
+		/**
+		 * Template for steps.
+		 *
+		 * @since 0.8.7
+		 */
+		public function print_templates() {
+			?>
+			<script type="text/html" id="tmpl-wc-api-guided-tour-step">
+				<div class="sf-guided-tour-step">
+					<a class="sf-guided-tour-close" href="#">
+						<span class="dashicons dashicons-no-alt"></span>
+					</a>
+					<# if ( data.title ) { #>
+						<h2>{{ data.title }}</h2>
+					<# } #>
+					{{{ data.message }}}
+					<# if ( data.altStep ) { #>
+						<a class="sf-nux-button sf-nux-alt-button {{ data.altStep.className }}" href="#">
+							{{ data.altStep.buttonText }}
+						</a>
+					<# } #>
+					<# if ( data.buttonText ) { #>
+						<a class="sf-nux-button sf-nux-primary-button {{ data.className }}" href="#">
+							{{ data.buttonText }}
+						</a>
+					<# } #>
+
+					<# if ( data.showSkip ) { #>
+						<a class="sf-guided-tour-skip" href="#">
+							<?php esc_attr_e( 'Skip this step', 'wc-api-dev' ); ?>
+						</a>
+					<# } #>
+				</div>
+			</script>
+			<?php
+		}
+
+		/**
+		 * Guided tour steps.
+		 *
+		 * @since 0.8.7
+		 */
+		public function guided_tour_steps() {
+			$steps = array();
+
+			$steps[] = array(
+				'title'       => __( 'Customize Your Store', 'storefront' ),
+				'message'     => __( 'It looks like your current theme isn\'t ready for shop features yet - your shop pages might look a little funny.</p><p>We suggest switching themes to <b>Storefront</b> which will bring out the best in your shop. Don\'t worry, if you try Storefront now, it won\'t be activated until you save your changes in the Customizer', 'storefront' ),
+				'buttonText'  => __( 'I\'ll keep my current theme', 'storefront' ),
+				'section'     => '#customize-info',
+				'className'   => 'sf-button-secondary',
+				'altStep'     => array(
+					'buttonText'  => __( 'I\'ll try Storefront!', 'storefront' ),
+					'action'      => 'expandThemes',
+					'message'     => __( 'Click the thumbnail to get started with Storefront', 'storefront' ),
+					'section'     => '#customize-control-theme_storefront .theme-screenshot',
+				)
+			);
+
+			// Determine what the next step should be
+			$needsLogo = ! has_custom_logo() && current_theme_supports( 'custom-logo' );
+			$logoBullet = $needsLogo ? __( '<li>Add your logo</li>', 'storefront' ) : '';
+
+			$steps[] = array(
+				'message'     => __( 'Okay! Remember you can switch themes at any time.</p><p>To get your store looking great, let\'s run through some common tasks:</p><ul>' . $logoBullet . '<li>Add Shop pages to your menus</li><li>Set your shop page as the homepage</li></ul><p>', 'storefront' ),
```

Comment:
The `$logoBullet` should be added to the string via placeholder/sprintf. Do you think translators might be confused because there's HTML but no opening `<p>` in some of these?

Reply:
Good call - I think there may be a few other localization issues in the PR too.